File: src/heartrate/HeartRate.java
package heartrate;

import java.util.Scanner;

public class HeartRate {

    public static void main(String[] args) {
        bpm();
    }

    private static void bpm() {
        String in = """
                2
                6 5.0000
                2 3.1222
                """;
        Scanner scan = new Scanner(in);
        int n = scan.nextInt();

        for (int i = 0; i < n; i++) {
            int beats       = scan.nextInt();
            double seconds  = scan.nextDouble();
            double BPM      = 60 * beats / seconds;
            double diff     = 60 / seconds;

            System.out.printf("%.4f %.4f %.4f \n", (BPM - diff), BPM, (BPM + diff));
        }
    }
}

File: src/heartrate/heartrate.go
package main

import (
    "bufio"
    "fmt"
    "os"
    "strconv"
    "strings"
)

func main() {
    io := bufio.NewScanner(os.Stdin)

    io.Scan()
    cases, _ := strconv.Atoi(io.Text())

    for i := 0; i < cases; i++ {
        io.Scan()
        line := strings.Fields(io.Text())
        beats, _ := strconv.ParseFloat(line[0], 10)
        secs, _ := strconv.ParseFloat(line[1], 10)

        BPM := 60 * beats / secs
        diff := 60 / secs

        fmt.Printf("%0.4f %0.4f %0.4f\n", BPM - diff, BPM, BPM + diff)
    }
}
